front/src/applications/operationalStudies/views/ManageTrainSchedule.tsx

```diff
@@ -74,7 +72,7 @@ export default function ManageTrainSchedule() {
     content: (
       <div className="row no-gutters">
         <div className="col-xl-6 pr-xl-2">
-          {mustUpdatePathfinding !== undefined && <Itinerary mustUpdate={mustUpdatePathfinding} />}
+          {mustUpdatePathfinding && <Itinerary mustUpdate={mustUpdatePathfinding} />}
```

Comment:
You need to remove the test here :)
```suggestion
          <Itinerary mustUpdate={mustUpdatePathfinding} />
```